addons/crm_iap_lead_enrich/__manifest__.py

```diff
@@ -14,6 +14,7 @@
         'data/ir_action.xml',
         'data/mail_data.xml',
         'views/crm_lead_views.xml',
+        'views/res_config_settings_view.xml',
```

Comment:
can't it be integrated in the above view? this way, no change of manifest required

Reply:
This way it's more organized after all. And AFAIK this change won't have effect unless you update, and will have if you do, so... Does it matter?